webpack.config.js

```diff
@@ -81,6 +81,7 @@ module.exports = (env) => {
     },
     plugins: removeEmpty([
       new HtmlWebPackPlugin({
+        favicon: './client/src/assets/images/favicons/favicon.ico',
```

Comment:
Can you change the favicon path to `./client/**public**/assets/images/favicons/favicon.ico'? It'll then use the _public_ folder that is output from webpack's bundling.

But before that we have to address building that _public_ folder. Did some research on webpack and realized that I wasn't actually bundling our app before running it. Don't think anyone else was either 🤷‍♂️.

So we need to add the bundling to our start command in `package.json`. There's multiple ways we could do this but let's keep it straightforward and change our `start:dev` script to `"webpack --env development && webpack serve --env development"`.

Then run `npm run start:dev` and the favicon should appear. You might need to clear cookies on `localhost:3000` as well.

Reply:
It worked! Thanks for doing all of the heavy lifting lol. I was off course trying to `npm run build:dev` to see if the webpack changes worked. I really appreciate you taking the time to both help and explain your logic! 

![Screenshot from 2022-10-11 14-36-50](https://user-images.githubusercontent.com/73561520/195203041-f5420be0-6cb5-49b7-9988-a189a888fa9b.png)